website/project/views/comment.py

```diff
@@ -17,30 +18,69 @@
 
 
 @file_updated.connect
-def update_comment_root_target_file(self, node, event_type, payload, user=None):
-    if event_type == 'addon_file_moved':
+def update_file_guid_referent(self, node, event_type, payload, user=None):
```

Comment:
This is a fairly hefty function, and the logic is fairly involved in the if statements. It might be a good idea to break it up a bit so that the logic is separate from the functionality.